.github/workflows/circuit.yml

```diff
@@ -0,0 +1,71 @@
+# This workflow will run tests using node and then publish a package to GitHub Packages when a release is created
+# For more information see: https://docs.github.com/en/actions/publishing-packages/publishing-nodejs-packages
+name: Node.js CI
+
+on:
+  pull_request:
+    branches:
+      - main                                  # when PR merge to main
+    paths :
+      - 'packages/circuits/**'                # when circuit file modified in PR
+      - 'packages/proof/**'                   # when proof file modified in PR
```

Comment:
should ci also run when `packages/contracts/**` or `jssdk` changes?

Reply:
circuit test do not rely on contract/jssdk.